app-emulation/wine/patches/wine-6.0~rc1.patchset

```diff
@@ -0,0 +1,358 @@
+From f29873c8684b673f71895d53532b2e2bc5a27d14 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 6 Dec 2020 22:57:13 +0100
+Subject: Add libnetwork to configure.ac
+
+
+diff --git a/configure.ac b/configure.ac
+index 020fd19..c0b393c 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -2201,6 +2201,9 @@ fi
+ dnl Check for -lnsl for Solaris
+ AC_SEARCH_LIBS(gethostbyname, nsl)
+ 
++dnl Check for -lnetwork for Haiku
++AC_SEARCH_LIBS(connect, network)
```

Comment:
you can combine the solaris check with the Haiku one
```suggestion
+AC_SEARCH_LIBS(connect, socket network)
```

Reply:
Thanks, thats true.